packages/common/package.json

```diff
@@ -28,7 +28,7 @@
     "url": "https://github.com/AudiusProject/audius-client/issues"
   },
   "dependencies": {
-    "@audius/sdk": "2.0.2-alpha.3",
+    "@audius/sdk": "2.0.2-alpha.4",
```

Comment:
i think you can ignore/restore these package json and package lock changes for future ref, but its fine since theyre already here